modules/caddyhttp/fileserver/caddyfile.go

```diff
@@ -58,6 +58,12 @@ func parseCaddyfile(h httpcaddyfile.Helper) (caddyhttp.MiddlewareHandler, error)
 				return nil, h.ArgErr()
 			}
 			fsrv.Browse = new(Browse)
+		case 2:
+			if args[0] != "browse" || args[1] != "reveal_symlink" {
+				return nil, h.ArgErr()
+			}
+			fsrv.Browse = new(Browse)
+			fsrv.Browse.RevealSymLink = true
```

Comment:
I think this option shouldn't be allowed inline, that's awkward. Instead, it should be a subdirective (like `browse` already is, taking a template file as an argument). Look at line 109.

Reply:
I just submitted an update to this. I am not sure if what I did is the best way to go so waiting on your feedback before marking this as resolved. Thanks again for reviewing.

Reply:
I think you should just add another `case "reveal_symlink"`, don't try to complicate `browse` itself.

Reply:
Okay, I think I finally got it  😅 . I didn't get that you did not want it to be an argument to browse.